example/panel/modules/data_load_async_module.dart

```diff
@@ -42,6 +44,8 @@ class DataLoadAsyncModule extends Module {
   @protected
   Future<Null> onLoad() {
     // trigger non-blocking async load of data
+    listenToStream(_events.didLoadData.take(1),
+        (_) => specifyStartupTiming(StartupTimingType.firstUseful));
```

Comment:
I don't think it really matters, but just for my own clarification, this could just be `specifyFirstUsefulState()` right?

Reply:
Yep, either way works. We weren't decided on whether we wanted shorthand at the time the example was written.